src/goto-instrument/contracts/dynamic-frames/dfcc.h

```diff
@@ -58,16 +58,16 @@ class optionst;
 
 // clang-format off
 #define HELP_DFCC                                                              \
-  "--dfcc <harness>   activate dynamic frame condition checking for function\n"\
-  "                   contracts using the given harness as entry point"
+  " --dfcc <harness>   activate dynamic frame condition checking for function\n"\
+  "                    contracts using the given harness as entry point\n"
```

Comment:
Thank you for spotting and fixing this, but since `cpplint` started complaining anyway: could you please insert more whitespace before the text description so that it aligns with the preceding options? Currently it looks like this:
```
                             force aggressive slicer to preserve all direct paths

Code contracts:
--dfcc <harness>   activate dynamic frame condition checking for function
```
The "activate" should align with "force". When doing so, please also adjust the line ends, even when this may require spreading the text over several more lines. (Eventually #6931 will take care of this automatically, but it might be a while until I get to cleaning it up.)

Reply:
Done. The "force" referred to was actually not aligned with previous options, so I aligned it as well.